src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1055,6 +1055,36 @@ def view_in_browser(resource_group_name, name, slot=None, logs=False):
         get_streaming_log(resource_group_name, name, provider=None, slot=slot)
 
 
+def view_appservice_samples_in_browser():
+    url = 'https://go.microsoft.com/fwlink/?linkid=864819'
+    _open_page_in_browser(url)
+
+
+def view_appservice_docs_in_browser():
+    url = 'https://go.microsoft.com/fwlink/?linkid=864822'
+    _open_page_in_browser(url)
+
+
+def view_webapp_samples_in_browser():
+    url = 'https://go.microsoft.com/fwlink/?linkid=864819'
+    _open_page_in_browser(url)
+
+
+def view_webapp_docs_in_browser():
+    url = 'https://go.microsoft.com/fwlink/?linkid=864822'
+    _open_page_in_browser(url)
+
+
+def view_functions_samples_in_browser():
+    url = 'https://go.microsoft.com/fwlink/?linkid=864823'
+    _open_page_in_browser(url)
+
+
+def view_functions_docs_in_browser():
+    url = 'https://go.microsoft.com/fwlink/?linkid=864821'
```

Comment:
1/2 of the time this directed me to a New York Times article about children's toys, and not the documentation (the other half, it did.) This makes it seem like `go.microsoft.com` is fragile. Is there another mechanism for linking out like this, maybe via `aka.ms`?

Reply:
We are trying to increase the usability and visibility of our docs and samples.
the main idea is to make them easy to discover and increase the usage of cli commands in the process.
@SyntaxC4 as he provided these fwlinks

Reply:
@sptramer we tried the fwlinks a number of times, we were never able to reproduce this issue